packages/adapters-library/src/adapters/snapshots.test.ts

```diff
@@ -321,3 +326,12 @@ async function loadJsonFile(
     rpcResponses,
   }
 }
+
+// This is to avoid having to update every snapshot when a new field is added to the response
+function sanitizeSnapshot(snapshot: AdapterResponse<unknown>[]) {
+  for (const position of snapshot) {
+    if (position.success) {
+      position.protocolDisplayName = ''
+    }
+  }
+}
```

Comment:
We can save ourselves having to update every single snapshot when we add a field like this.